### ISSUE TITLE
Plugin doesn't work after OS update

### ISSUE THREAD

**Issue by keywok** [1Defence/client-anti-drag#3]
Moved from W10 to W11 and since the plugin doesn't stop the client from being instantly draggable. Any further information that would be needed let me know.

Client Version: 1.10.22
Plugin Version: 2081a9ed
Windows Version: 22621.3155

**Comment by questionabl3** [comment]
yeah i liked this plugin, hope it gets updated

**Comment by KingJoey21** [comment]
Yes please update as soon as you get some free time. This client has been huge QoL for me

**Comment by 1Defence** [comment]
I've missed these issues but unfortunately it's in a permanent state of unfixable until core changes are possibly made in future. The blocked drag event no longer exists after the Flatlaf update as runelite migrated to letting the windows OS handle the window events.
